package.json

```diff
@@ -9,6 +9,7 @@
     "@types/react-dom": "^16.9.0",
     "@types/react-instantsearch-dom": "^6.3.0",
     "@types/react-router-dom": "^5.1.5",
+    "@types/styled-components": "^5.1.2",
```

Comment:
Why do you need to pull in styled-components? We're not using them for our components. You should be able to use external saas files.

Reply:
done!